solutions/remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -1373,6 +1373,7 @@
                                     "inputs": {
                                         "body": {
                                             "Body": "@body('Parse_JSON')?['body']",
+                                            "Importance": "Normal",
```

Comment:
What was default w/o this value?

Reply:
There is a bug where it was defaulting to "Low Importance"